### PR TITLE
Cleaning up the readme and TravisCI configuration

### PR DESCRIPTION
**Pull request by acant**
* add recent Ruby versions to TravisCI
* add Elsewhere section to readme
* bring the badges up to date
* add consistent contribution guidelines
* update the copyright marker

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,20 +5,35 @@ rvm:
   - 2.3
   - 2.4
   - 2.5
+  - 2.6.2
   - ruby-head
+  - jruby
```
Comment (sugarb) [inline .travis.yml:10]: Should this really be using jruby? Or does this test with all the rubies listed?

Comment (acant) [reply .travis.yml:10]: Yes this test with all the Rubies listed. The idea being to try to see where the gem will run. The places which are not expected to always succeed are marked as 'allow_failure' so the will not fail the build. (i.e., jruby and ruby-head)

**Review by dhuang-sugarcrm — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -5,20 +5,35 @@ rvm:
   - 2.3
   - 2.4
   - 2.5
+  - 2.6.2
```
Comment (yads) [inline .travis.yml]: Do we want a specific 2.6 version here, or just 2.6?

**Review by yads — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (acant) [reply .travis.yml]: Yes, I just want the latest 2.6. I don't remember why I was this specific.
I will try using 2.6 and see if that will do the job.

FYI The travis ruby list is here is anyone wants to see what I am working from
http://rubies.travis-ci.org/

Comment (acant) [reply .travis.yml]: 2.6 works just fine. I don't know why I was more specific. 🤷‍♂

**Pull request merged**